src/loaders/svg/tvgSvgCssStyle.cpp

```diff
@@ -28,17 +28,30 @@
 /* Internal Class Implementation                                        */
 /************************************************************************/
 
+static bool _checkImportance(SvgStyleFlags &toFlagsImportance, SvgStyleFlags fromFlagsImportance, SvgStyleFlags flag)
```

Comment:
intuitive naming:
_checkImportance() -> _notImportance()

Reply:
```checkImportance``` checks whether ```from``` has set the ```importance``` tag for a given property and whether ```to``` hasn't. so it's neither ```yes``` (important) or ```not``` (not important) answer. 
for me ```_notImportance``` isn;t an intuitive name, but if you think it's a better name I'll change it.

Reply:
ok, I'm confused what it does.